Pikipick/Pikipick/ButtonAnimView.swift

```diff
@@ -48,96 +60,93 @@ struct ButtonAnimView: View {
     }
 }
 
+//ref: https://www.youtube.com/watch?v=S7hhHc9FgnY&t=639s
 struct ConfettiContainer: View {
-    @State var movement = Movement(x: 0, y: 0, z: 1, opacity: 0)
+    @State var movement = Movement(x: 0, y: 0, emojiSize: 0.1, opacity: 0)
+    @State var animationValues: [Bool] = Array(repeating: false, count: 5)
+    @State var emojiPositionSet = false
     @Binding var animate:Bool
```

Comment:
```suggestion
    @Binding var animate: Bool
```
공백 추가로 스타일을 맞춰주는 센스!

Reply:
공백 추가 스타일 맞춰주는센스!!